### PR TITLE
Feature/delayed email

### PR DESCRIPTION
**Pull request by Ahana15**
An email will be sent to the user after 20 secs, If a user has registered for the first time and directed to the settings page but hasn't clicked the dashboard button and closed the browser instead.

However, there is a small bug, the email is sent twice as the browser close event is somehow being fired twice. (not sure if you guys can find a fix for this)

### CODE REVIEW

File: server/services/emails/email.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const Queue = require("bull");
+const Arena = require("bull-arena");
+const sgMail = require("@sendgrid/mail");
+require("dotenv").config();
```
Comment (derekjou) [inline server/services/emails/email.js:4]: If we're using this file in server.js with an app.use(), we won't need this line

File: client/src/pages/Settings.js [hunk 2]
```diff
@@ -90,6 +94,14 @@ const Settings = () => {
...
   };
 
+  window.addEventListener(
+    "beforeunload",
+    event => {
+      axios.post(`/queue/${localStorage.getItem("email")}`);
+    },
+    false
+  );
+
```
Comment (derekjou) [inline client/src/pages/Settings.js]: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener 

Maybe try using the options object, setting "once" to true?

Comment (Ahana15) [reply client/src/pages/Settings.js]: changed it to true, and it seems to be being fired once now. THANK YOU!

File: client/src/pages/Settings.js [hunk 3]
```diff
@@ -39,39 +39,43 @@ const Settings = () => {
...
+    );
     setSearch("");
-  }
+  };
 
-  const onSearchChange = (event) => {
+  const onSearchChange = event => {
     setSearch(event.target.value);
-  }
+  };
```
Comment (ncen5293) [inline client/src/pages/Settings.js]: Should probably undo these changes since I put the functions for the searchbar in the searchbar component on my pr so this would just add in unneeded functions
